src/wp-includes/blocks.php

```diff
@@ -1174,12 +1174,33 @@ function parse_blocks( $content ) {
  */
 function do_blocks( $content ) {
 	$blocks = parse_blocks( $content );
+
+	/**
+	 * Filter that allows plugins to inspect or manipulate the array of parsed
```

Comment:
Docblocks should always begin with a third-person singular verb, so "Filters" rather than "Filter". See [the handbook entry](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language).

Reply:
But isn't this referring to the filter itself, and not the action of filtering?

By the way, I was just renaming them to this, following [Dennis request](https://github.com/WordPress/wordpress-develop/pull/5023#pullrequestreview-1633871328):

- `Filter to allow inspection and modification of the tree of raw parsed blocks before they are rendered.`
- `Filter to allow processing the final HTML output from block rendering.`

Reply:
Core uses the same language for functions, methods, filters and actions, in that it describes **what it does**, rather than **what it is**.

Reply:
You can see this example in the handbook entry I linked above:

> **Filters:** What is being filtered?
> Good: Filters the post content.
> Bad: Lets you edit the post content that is output in the post template.

Reply:
[Nevermind](https://github.com/WordPress/wordpress-develop/pull/5023#discussion_r1340405194) 👍

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5023/commits/7df954bf3eb0f317681e52e7cbaca6c551244244.

Reply:
My bad. I was using Core's own code as inspiration - a terrible flaw 😆